build/bin/network-validator.go

```diff
@@ -20,7 +20,7 @@ import (
 )
 
 var (
-	timeout        = flag.Duration("timeout", 1000*time.Millisecond, "Timeout for each dial request made")
+	timeout        = flag.Duration("timeout", 2000*time.Millisecond, "Timeout for each dial request made")
```

Comment:
Looks like this value matches the default for the var. Slight chance of the two falling out of sync. Any chance this could be a central config value? And use that const as default in cmd as well as this?

Reply:
yes, This parameter also exists in https://github.com/openshift/osd-network-verifier/blob/main/pkg/cloudclient/cloudclient.go#L28 and allows us to pass it to `network-validator.go`'s dockerised form as to be run by `userdata` in bootstrap step of the ec2 instance.

I think that it makes sense to set it to a time greater than 1

Reply:
Ok